tracing-subscriber/src/fmt/mod.rs

```diff
@@ -873,6 +873,15 @@ impl<N, E, F, W> SubscriberBuilder<N, E, F, W> {
             inner: self.inner.with_writer(make_writer),
         }
     }
+
+    /// Prints text using the standard library's macro `print!`. This writer allows stdout logs to
+    /// be correctly captured by commands such as `cargo test`.
```

Comment:
```suggestion
    /// A writer intended to be used in unit tests. See [TestWriter] for additional details.
```

...where [TestWriter] is a valid doc link.

Reply:
Since this is a method on the builder, I think the doc comment should begin with "Configures the subscriber to use..." or similar?

Maybe:

```suggestion
    /// Configures the subscriber to support [`libtest`'s output capturing][capturing]
    /// when used in unit tests.
    ///
    /// This sets the writer used by the subscriber being constructed to a [`TestWriter`],
    /// which prints text using the standard library's `print!` macro. This writer allows stdout
    /// logs to be correctly captured by commands such as `cargo test`.
    ///
    /// [capturing]: https://doc.rust-lang.org/book/ch11-02-running-tests.html#showing-function-output
    /// [`TestWriter`]: writer/struct.TestWriter.html
```

And, an example would definitely also be nice if you're up for it! :)